### ISSUE TITLE
Redirect people to Field Papers?

### ISSUE THREAD

**Issue by almccon** [migurski/paperwalking#53]
Maybe just a link on the main Walking Papers page to let people know they should try Field Papers instead?

**Comment by migurski** [comment]
I would love to do this. A while back I asked Seth about the possibility of migrating people's Walking Papers content to Field Papers so that we could do a complete redirect instead of a message. It would require some work so we didn't end up doing it. Would that be a possibility?

**Comment by almccon** [comment]
I haven't looked into the difficulty of doing that, but it's hard to imagine when we'd have budget to do the migration. Are there any important users of WP who would fund that work as part of a future grant? Or even just people who could vouch for why it's important work that we need to do? Since we haven't done it already, it's going to be hard to make the case that we need to migrate over the old WP maps that are mostly over two years old.

**Comment by migurski** [comment]
I don't really have useful answers to your questions, I'm sorry. I'm not seeking grant funding for WP, and I think all the actual users are with FP. Sounds like the only real change that can happen is a note about the move somewhere on the site.

**Comment by almccon** [comment]
Yeah, I think just a note somewhere on the site is probably all we can do at this point.

**Comment by migurski** [comment]
Sounds good!
